api/tests/routes/pro/delete_stocks_test.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+import pcapi.core.offerers.factories as offerers_factories
+import pcapi.core.offers.factories as offers_factories
+import pcapi.core.offers.models as offer_models
+import pcapi.core.users.factories as users_factories
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns204Test:
+    def test_delete_multiple_stocks_by_offer_id(self, client):
+        # Given
+        user = users_factories.UserFactory()
+        offer = offers_factories.OfferFactory()
+        offerers_factories.UserOffererFactory(user=user, offerer=offer.venue.managingOfferer)
+
+        batch_stocks = offers_factories.StockFactory.create_batch(3, offer=offer)
+        data = {"ids_to_delete": [stock.id for stock in batch_stocks]}
+
+        # When
+        response = client.with_session_auth(user.email).post(f"/offers/{offer.id}/stocks", json=data)
+
+        # Then
+        assert response.status_code == 204
+
+        assert offer_models.Stock.query.first().isSoftDeleted
+
+    def test_delete_unaccessible_stocks(self, client):
+        # Given
+        offer = offers_factories.OfferFactory()
+        another_offer = offers_factories.OfferFactory()
+        user = users_factories.UserFactory()
+        offer = offers_factories.OfferFactory()
+        offerers_factories.UserOffererFactory(user=user, offerer=offer.venue.managingOfferer)
+
+        batch_stocks = offers_factories.StockFactory.create_batch(3, offer=another_offer)
+        data = {"ids_to_delete": [stock.id for stock in batch_stocks]}
+
+        # When
+        response = client.with_session_auth(user.email).post(f"/offers/{offer.id}/stocks", json=data)
```

Comment:
j'ai trouvé le test un peu difficile à lire, notamment pourquoi on ne renvoit pas de 404 pour les stocks d'`another_offer`, mais c'est parce qu'on utilise `offer.id` et pas `another_offer.id`

je trouverais ce test un peu plus lisible
```suggestion
        another_offer_stocks = offers_factories.StockFactory.create_batch(3, offer=another_offer)

        # When
        response = client.with_session_auth(user.email).post(f"/offers/{offer.id}/stocks", json={"ids_to_delete": [stock.id for stock in another_offer_stocks]})
```